### PR TITLE
Sc 15400/invite page

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds views for new invited users and existing invited users to Ensign. `RegistrationForm` is currently used for the existing user page and a new form has been created for new invited users.

This was mostly copied over from the current registration and login pages and will need to be updated once the `Login` and `Register` endpoints are updated. Additional stories will be created to update the logic and for e2e tests.

Fixes SC-15400

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Figma designs: https://www.figma.com/file/PvL23fm6W36Zd88O1XVyIo/Beacon-DX-UI?node-id=4603%3A10760&t=1PQbFp1FmXBmFlml-1

New Invited User Page: https://www.awesomescreenshot.com/video/16323388?key=934c2f8bae67ae7c4a44baf23c7d7f2e

Existing Invited User Page: https://www.awesomescreenshot.com/video/16323436?key=9ba213751897fa5f94382b0f20691859

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/members/utils.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { formatDate } from '@/utils/formatDate';
 export const formatMemberData = (m: any) => {
-  console.log('[formatMemberData] m: ', m);
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/members/utils.ts:3]: Would it be better to comment this out instead of deleting?

Comment (elysee15) [reply web/beacon-app/src/features/members/utils.ts:3]: I think yes

File: web/beacon-app/src/features/teams/components/RegisterNewUser/NewInviteRegistrationForm.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                      fullWidth
+                      {...getFieldProps('password')}
+                      onFocus={onFocus}
+                      onBlur={onBlur}
+                    />
+                  </div>
+                </RadixTooltip.Trigger>
+                <RadixTooltip.Portal>
+                  <RadixTooltip.Content
+                    className="select-none rounded-[4px] bg-white px-[15px] py-[10px] text-xs text-[15px] leading-none shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] will-change-[transform,opacity] data-[state=delayed-open]:data-[side=top]:animate-slideDownAndFade data-[state=delayed-open]:data-[side=right]:animate-slideLeftAndFade data-[state=delayed-open]:data-[side=left]:animate-slideRightAndFade data-[state=delayed-open]:data-[side=bottom]:animate-slideUpAndFade"
```
Comment (elysee15) [inline web/beacon-app/src/features/teams/components/RegisterNewUser/NewInviteRegistrationForm.tsx:96]: I think those classnames already exist in the `tailwind.config.ts`

**Review by elysee15 — APPROVED** [review]

File: web/beacon-app/src/features/teams/components/RegisterNewUser/NewInviteRegistrationForm.tsx (hunk 2 above)
Comment (daniellemaxwell) [reply web/beacon-app/src/features/teams/components/RegisterNewUser/NewInviteRegistrationForm.tsx:96]: I saw the class names that come after `animate` in the Tailwind config file. I wanted the style to agree with the `Registration Form` we have now, so I used the same class names I saw in that file.

**Pull request merged**